js/client/modules/@arangodb/testutils/testing.js

```diff
@@ -229,6 +230,7 @@ const optionsDefaults = {
   'valgrindArgs': {},
   'valgrindHosts': false,
   'verbose': false,
+  'noStartStopLogs': false,
```

Comment:
Please make this dependend on whether its an interactive shell or not like:
```
  'setInterruptable': ! internal.isATTy(),
```

Reply:
Done.